src/text_engine/effects/WipeInventory.java

```diff
@@ -0,0 +1,26 @@
+package text_engine.effects;
+
+import text_engine.characters.PlayerCharacter;
+
+/**
+ * Represents an effect on the player. Affects anything about the player's status.
+ */
+public class WipeInventory extends Effect<PlayerCharacter> {
+
+    @Override
+    public void accept(Object o) {
+        PlayerCharacter p = (PlayerCharacter) o;
+        p.wipeInventory();
```

Comment:
If this is going to be an effect, then this convenience method is only going to be used here. Just use `p.getInventory().clear()`.